packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -405,6 +426,29 @@ class Auth {
         this.clearStorage()
         return this.loginGuestUser()
     }
+
+    /**
+     * Decode SLAS JWT and extract information such as customer id, usid, etc.
+     *
+     */
+    decodeSlasJWT(jwt: string) {
+        const payload = jwtDecode(jwt) as SlasJwtPayload
+        const {sub, isb} = payload
+        // ISB format
+        // 'uido:ecom::upn:Guest||xxxEmailxxx::uidn:FirstName LastName::gcid:xxxGuestCustomerIdxxx::rcid:xxxRegisteredCustomerIdxxx::chid:xxxSiteIdxxx',
+        const isGuest = isb.split('::')[1].replace('upn:', '') === 'Guest'
+        const customerId = isGuest
+            ? isb.split('::')[3].replace('gcid:', '')
+            : isb.split('::')[4].replace('rcid:', '')
```

Comment:
```suggestion
        const isbParts = isb.split('::')
        const isGuest = isbParts[1] === 'upn:Guest'
        const customerId = isGuest
            ? isbParts[3].replace('gcid:', '')
            : isbParts[4].replace('rcid:', '')
```
1. Save the split result as a variable.
2. You don't really need string manipulation if you're checking a hard-coded value.

Reply:
good stuff